packages/atomic-angular/projects/atomic-angular/src/public-api.ts

```diff
@@ -3,4 +3,5 @@
  */
 export * from '@coveo/atomic/headless';
 export * from './lib/stencil-generated/components';
+export {Bindings, i18n} from '@coveo/atomic';
```

Comment:
These interfaces weren't exported by `@coveo/atomic-angular` and `@coveo/atomic-react`, which means that we could only import from from `@coveo/atomic`, which gives a [`no-extraneous-import`](https://github.com/mysticatea/eslint-plugin-node/blob/v11.1.0/docs/rules/no-extraneous-import.md) error with the recommended ESLint config.